src/dms/clis/EdFi.DataManagementService.SchemaGenerator.Tests.Unit/Engines/Mssql/__snapshots__/MssqlFlatteningMetadataComprehensiveTests.ValidateDescriptorForeignKeyTarget_PrefixedTables.snap

```diff
@@ -0,0 +1,36 @@
+﻿-- MSSQL CREATE TABLE edfi_edfi_AssessmentItem
+IF NOT EXISTS (SELECT * FROM sys.tables WHERE name = 'edfi_edfi_AssessmentItem' AND SCHEMA_NAME(schema_id) = 'dms')
```

Comment:
Double edfi in table names, I think you just need to remove `edfi_*` from the table BaseNames in the tests